lisp/forge-github.el

```diff
@@ -491,6 +507,48 @@
                  (forge--pull repo nil cb))))
     (funcall cb)))
 
+(cl-defmethod forge--fetch-workflow ((repo forge-github-repository) pr-id callback)
+  (ghub--graphql-vacuum
+   '(query
+     (node [(id $pr ID!)]
+           (\...\ on\ PullRequest
+            id
+            (commits [(last 1)]
+                     (edges
+                      (node
```

Comment:
Instead of `(commits (edges (node` use `(commits [:edges t]`. That departs from vanilla graphql but looks cleaner and also makes it easier to process the returned data because one unnecessary nesting level is automatically removed.

Unfortunately I haven't written any documentation yet (and don't use this often enough not to have to look at existing code to remember how to do things). I do plan to write some documentation, but I am afraid it will be too late for you.

Reply:
great to know about. also solves having to worry about pagination